Remotion/Web/Development.WebTesting/CoypuElementScopeExtensions.cs

```diff
@@ -249,11 +249,11 @@ public static void Unhover ([NotNull] this ElementScope scope)
     }
 
     /// <summary>
-    /// Gets an attribute from the scope or throws a <see cref="MissingHtmlException"/> if it could not be found. Wraps <see cref="ElementScope.this"/>.
+    /// Gets the specified HTML attribute's value from the <paramref name="scope"/>.
     /// </summary>
     /// <remarks>
     /// Accessing HTML attributes through <see cref="ElementScope.this"/> returns null if the attribute does not exist.

```

Comment:
the see langword=null is missing.

Reply:
Fixed in https://github.com/re-motion/Trunk-SVN2Git/pull/11/commits/bfbe4f17df1e9f7458bdce8943ec30c063b879d9.